conda/recipes/libcudf/recipe.yaml

```diff
@@ -20,6 +20,15 @@ cache:
   build:
     script:
       content: |
+
+        # Remove `-fdebug-prefix-map` line from CFLAGS and CXXFLAGS so the
```

Comment:
What adds these flags?

Reply:
both `conda-build` and `rattler-build` add them in this file in the build environment:
`/etc/conda/activate.d/activate-gxx_linux-64.sh`

There's a line in there like `-fdebug-prefix-map=${SRC_DIR}=/usr/local/src/conda/${PKG_NAME}-${PKG_VERSION}`

but `conda-build` uses the recipe name and leaves `$PKG_VERSION` unset, so it doesn't bust the cache.

upstream issue opened to fix this: https://github.com/prefix-dev/rattler-build/issues/1458

but it might take a bit of time to get a fix so adding the workaround here to get us unblocked.